net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml

```diff
@@ -3,6 +3,7 @@
     <General VisibleName="General" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/general/index" order="1"/>
     <RIP VisibleName="RIP" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/rip/index" order="10" />
     <OSPF VisibleName="OSPF" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf/index" order="20" />
+    <OSPF6 VisibleName="OSPFv3" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf6/index" order="25" />
```

Comment:
we need to find something other than fa-bolt for these items (not just the addition)... it is typically used for proxy software in our menu and it looks weird if we only have the same icon all the time. that's purely cosmetic, but can go a long way. I guess I should work on this separately.